### PR TITLE
[VideoToolbox] Update to Xcode 9.3 Beta 1

### PR DESCRIPTION
**Pull request by chamons**
- https://github.com/xamarin/xamarin-macios/wiki/VideoToolbox-iOS-xcode9.3-beta1

### CODE REVIEW

File: src/videotoolbox.cs [hunk 1]
```diff
@@ -625,6 +625,14 @@ interface VTDecompressionPropertyKey {
 
 		[Field ("kVTDecompressionPropertyKey_PixelTransferProperties")]
 		NSString PixelTransferProperties { get; }
+
+		[Mac (10, 13)]
+		[Field ("kVTVideoDecoderSpecification_RequiredDecoderGPURegistryID")]
+		NSString RequiredDecoderGPURegistryID { get; }
```
Comment (dalexsoto) [inline src/videotoolbox.cs]: GPU -> Gpu
ID -> Id (not an acronym)

`RequiredDecoderGpuRegistryId`

File: src/videotoolbox.cs [hunk 2]
```diff
@@ -625,6 +625,14 @@ interface VTDecompressionPropertyKey {
...
+
+		[Mac (10, 13)]
+		[Field ("kVTVideoDecoderSpecification_PreferredDecoderGPURegistryID")]
+		NSString PreferredDecoderGPURegistryID { get; }
```
Comment (dalexsoto) [inline src/videotoolbox.cs]: GPU -> Gpu
ID -> Id (not an acronym)

`PreferredDecoderGpuRegistryId`

File: src/videotoolbox.cs [hunk 3]
```diff
@@ -625,6 +625,14 @@ interface VTDecompressionPropertyKey {
 
 		[Field ("kVTDecompressionPropertyKey_PixelTransferProperties")]
 		NSString PixelTransferProperties { get; }
+
+		[Mac (10, 13)]
```
Comment (dalexsoto) [inline src/videotoolbox.cs]: Are these Mac only? if so you need `NoTV` and `NoiOS`

**Comment by chamons** [comment]
I totally missed sibling keys on VTVideoDecoderSpecificationKeys class that these should go with..

**Comment by dalexsoto** [comment]
build

**Comment by dalexsoto** [comment]
issuing new build so we get xtro report.

File: tests/xtro-sharpie/tvOS-VideoToolbox.ignore [hunk 4]
```diff
@@ -0,0 +1,2 @@
+!missing-field! kVTVideoDecoderSpecification_PreferredDecoderGPURegistryID not bound
```
Comment (spouliot) [inline tests/xtro-sharpie/tvOS-VideoToolbox.ignore]: minor: `git blame ` can help but it's easier (in `.ignore` files) when a comment like `### API not marked with unavailable (but without availability for tvOS/watchOS)` is added

**Pull request merged**